### PR TITLE
EZP-27443: Display main content tab

### PR DESCRIPTION
**Pull request by maxbaldanza**

### CODE REVIEW

File: src/lib/Repository/Values/Content/ContentType.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType as APIContentType;
+
+/**
+ * Adds a new method to content type that allows us to provide extra functionality.
+ */
+class ContentType extends \eZ\Publish\Core\Repository\Values\ContentType\ContentType
```
Comment (mcurran-inviqa) [inline src/lib/Repository/Values/Content/ContentType.php]: rename to UiContent type then can import orig

File: src/lib/Repository/Values/Content/ContentType.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{
+    public function __construct(APIContentType $contentType)
```
Comment (mcurran-inviqa) [inline src/lib/Repository/Values/Content/ContentType.php]: probably worth adding in additional properties for future extension/consistenct with others.

File: src/bundle/Resources/views/content/tabs/content.html.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% set languageCode = content.versionInfo.contentInfo.mainLanguageCode %}
+
+{% for contentTypeGroup in contentType.contentTypeGroups %}
+    {% set contentTypeName = contentTypeGroup.name(languageCode) %}
+
+    <section class="ez-fieldgroups ez-fieldgroup ez-fieldgroup">
+        <h3 class="ez-fieldgroup-name"><a href="#" class="ez-collapse-toggle">{{ contentTypeName }}</a></h3>
+
+        {% for fieldDefintion in contentType.fieldDefinitionsByFieldGroup(contentTypeName) %}
```
Comment (mcurran-inviqa) [inline src/bundle/Resources/views/content/tabs/content.html.twig]: fieldDefinition

File: src/bundle/Resources/config/services.yml [hunk 4]
```diff
@@ -76,3 +76,7 @@ services:
     EzSystems\HybridPlatformUi\Repository\DecoratedLocationService:
         arguments:
             - "@ezpublish.api.service.location"
+
+    EzSystems\HybridPlatformUi\Repository\ContentTypeService:
+        arguments:
+            - "@ezpublish.signalslot.service.content_type"
```
Comment (mcurran-inviqa) [inline src/bundle/Resources/config/services.yml]: should this be ezpublish.api.service.content_type?

**Review by mcurran-inviqa — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/config/services.yml (hunk 4 above)
Comment (maxbaldanza) [reply src/bundle/Resources/config/services.yml]: They both map to the same thing but I presume the one you mention is more flexible and is the one to use 👍

**Pull request closed without merging**